lua/telescope/_extensions/packer.lua

```diff
@@ -30,6 +30,30 @@ local setup = function(opts)
   end
 end
 
+local function opener()
+  local sysname = vim.loop.os_uname().sysname
+  if sysname == "Darwin" then
+    return 'open'
+  elseif sysname == "Windows" then
+    return "explorer"
+  elseif vim.fn.executable("wslview") then
+    return 'wslview'
+  elseif vim.fn.executable("xdg-open") then
+    return 'xdg-open'
+  else
+    return nil
+  end
+end
+
+local function get_url(path)
+  local url = vim.fn.trim(vim.fn.system(string.format("git -C %s ls-remote --get-url", path)))
+  if string.sub(url, 1, 6) == 'ssh://' then
+    url, _ = string.gsub(url, '^ssh://%w+@', 'https://')
+    url, _ = string.gsub(url, '^ssh://', 'https://')
+  end
```

Comment:
oh one more thing it could also start with `git@github.com:nvim-telescope/telescope.nvim.git` e.g.